templates/membership-card.php

```diff
@@ -142,6 +142,7 @@
 						echo "Never";
 				?>
 			</p>
+			<?php do_action("pmpro_membership_card_additional_fields");?>

```

Comment:
Suggestion to pass through the `$pmpro_membership_card_user` so user data can be available, worked out well for me :)

Maybe we could add in a hook to the top of the fields as well?

Reply:
Revising my initial thoughts on this...Instead of just having a hook...I think we should add a filter for all the fields that will get displayed. It would be an array and include "Member Since", "Membership Level", and "Membership Expires" by default and follow a label => value format. Would allow for sorting, removing default fields, and adding additional fields.

Reply:
Yea that would be good too!